lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -55,6 +55,14 @@ static Value mapOperand(Value operand, Operation *op,
       // Cast firrtl -> standard type.
       return rewriter.create<firrtl::StdIntCast>(op->getLoc(), intType,
                                                  operand);
+    } else if (auto flippedType = resultType.getValue().dyn_cast<FlipType>()) {
```

Comment:
It might work better to just do a "opType = operand.getType().getPassiveType();" at the top, that way the integer (and other future cases) won't have to be duplicated.

Reply:
I like this idea, but it did not compile. 
`error: ‘class mlir::Type’ has no member named ‘getPassiveType’;`

Works on line 50, though. Thanks for the simplifying suggestion!